modelexpress_client/python/modelexpress/load_strategy/model_streamer_strategy.py

```
@@ -23,28 +27,67 @@
+
+    try:
+        from huggingface_hub import scan_cache_dir
+        cache_dir = os.environ.get("HF_HUB_CACHE") or os.environ.get("HF_HOME")
+        if cache_dir:
+            cache_info = scan_cache_dir(cache_dir)
+            for repo in cache_info.repos:
+                if repo.repo_id == uri:
+                    rev = max(repo.revisions, key=lambda r: r.last_modified)
+                    logger.info(f"Resolved HF model '{uri}' -> {rev.snapshot_path}")
+                    return str(rev.snapshot_path)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)